docs/ProgrammingCryptol/highAssurance/HighAssurance.tex

```diff
@@ -174,6 +174,51 @@ \subsection{Capturing test vectors}
   sure the tweak to the above example (removal of dummy single formal
   parameter) works.}
 
+%~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+\subsection{Generating test vectors}
+\label{sec:gentestvec}
+
+Cryptol has the ability to generate test vectors for simple function types using the \texttt{:dumptests} REPL command.
+When \texttt{:dumptests} is given a file to dump the output and a function in scope, it will randomly generate inputs for the function and run them for you.
+The output will contain a tab delimited file in which the first column contains the output, and each column after contains the inputs to the function.
+The command \texttt{:dumptests} uses the same settings as \texttt{:check} \ref{sec:quickcheck}, this means you can set the number of tests you want using the \texttt{:set tests=100} command.
+
+The following is an example using \href{https://github.com/GaloisInc/cryptol-specs/blob/7468d170f5e9e3913cd12e29c76d8c041ef99dd0/Primitive/Keyless/Hash/SHA2/Instantiations/SHA256.cry}{SHA2-256 implemntation} in \href{https://github.com/GaloisInc/cryptol-specs}{cryptol-specs}.
+The output file was modified slightly by putting the input on the next line tabbed in. 
+\begin{Verbatim}
+Cryptol> :load Primitive/Keyless/Hash/SHA2/Instantiations/SHA256.cry
+Loading module Cryptol
+Loading module `where` argument of Primitive::Keyless::Hash::SHA2::Instantiations::SHA256
+Loading interface module `parameter` interface of 
+  Primitive::Keyless::Hash::SHA2::Specification
+Loading module Primitive::Keyless::Hash::SHA2::Specification
+Loading module Primitive::Keyless::Hash::SHA2::Instantiations::SHA256
+Primitive::Keyless::Hash::SHA2::Instantiations::SHA256> :dumptests output.txt hash`{256}
+Primitive::Keyless::Hash::SHA2::Instantiations::SHA256> :quit
+% head output.txt                                                        
+0x6055ba8ba7ae5a07b972f621b0a1953e8769a8fba9e767b70016fe00b8173c61
```

Comment:
suggestion: This aligns with my other comment but it's hard to tell what's tab-separated and what's on separate lines with such long values. Can you maybe parameterize the function with some much smaller number so that the PDF renders the output on one line? Or use a different function with littler inputs?